crates/ra_hir/src/ty/tests.rs

```diff
@@ -168,15 +168,18 @@ fn test() {
     let x = a && b;
     let y = true || false;
     let z = x == y;
-    let h = CONST_1 <= CONST_2;
+    let minus_forty: isize = -40isize;
+    let h = minus_forty <= CONST_2;
     let c = f(z || y) + 5;
     let d = b;
-    let e = 3i32 && "hello world";
+    let g = minus_forty ^= i;
+    let ten: usize = 10;
+    let ten_is_eleven = ten == some_num;
 
-    10 < 3
+    ten < 3
 }
 "#,
-        "boolean_op.txt",
+        "binary_op.txt",
```

Comment:
it seems you forgot to add the renamed file?

Reply:
uhh, I forgot to add it to git I think! :fearful: